util/install_linux.sh

```diff
@@ -3,11 +3,14 @@
 # remove all fonts from ~/.local/share/fonts that start with "Monaspace"
 rm -rf ~/.local/share/fonts/Monaspace*
 
+# make the folder if it does not exist
+mkdir -p ~/.local/share/fonts 
+
 # copy all fonts from ./otf to ~/.local/share/fonts
-cp ./fonts/otf/* ~/.local/share/fonts
+cp ../fonts/otf/* ~/.local/share/fonts
```

Comment:
This depends on the current working directory when the script is called.

Maybe better use a path relative to the script location, then one can call the script from any directory?
```suggestion
# Get script directory
sd=$(cd -- "$(dirname "$0")" >/dev/null 2>&1 || exit; pwd)

# copy all fonts from ./otf to ~/.local/share/fonts
cp "${sd}/../fonts/otf/*" ~/.local/share/fonts
```

(Well, something to that effect)

Reply:
I think this has still not been addressed.

You could call `install.sh` from any current working directory. But all the paths in the script are relative to the current working directory instead of relative to the script directory. That in effect means one needs to change into the concrete directory that the author of the script envisioned.

For example
```console
cd ~/git/monaspace
util/install.sh

cd ~/git/monaspace/util
./install.sh

cd /tmp
~/git/monaspace/util/install.sh
```
They should imho all work.

For the script to assemble the source path of the font files it needs the absolute path of the script itself and from then on it could relative walk to the fonts directory. Thus the code above.